app/utils/renderBodyHandler.js

```diff
@@ -0,0 +1,25 @@
+const { STATUS_TYPE, STATUS_MESSAGE } = require('./statusCodes');
+
+class RenderBodyHandler {
```

Comment:
What's the purpose of this renderBody handler? Not seen any usage in your code!

Reply:
This file is not used in new version, I will delete it.